tests/subsys/llext/simple/src/test_llext_simple.c

```diff
@@ -417,9 +417,12 @@ ZTEST(llext, test_find_section)
 	uintptr_t symbol_ptr = (uintptr_t)llext_find_sym(&ext->exp_tab, "number");
 	uintptr_t section_ptr = (uintptr_t)find_section_ext + section_ofs;
 
-	zassert_equal(symbol_ptr, section_ptr,
+	/* on RISC-V, at least for GCC, the symbols aren't always at the beginning of the section */
+	if (IS_ENABLED(CONFIG_RISCV)) {
+		zassert_equal(symbol_ptr, section_ptr + section_padding,
```

Comment:
This now checks RISCV  _only_... :slightly_smiling_face: 
Also, `section_padding` looks undefined. Maybe have it default to 0 for non-RISCV targets?

Reply:
You are right, I wanted to test this on everything BUT RISC-V. Also, I will remove the padding.

Reply:
This problem also only occured with `LLEXT_TYPE_ELF_OBJECT`; I have removed the condition and added a comment stating that this will break if support for `LLEXT_TYPE_ELF_OBJECT` is ever added again.